mysql-test/columnstore/bugfixes/mcol-5992.test

```diff
@@ -0,0 +1,17 @@
+# MCOL-5992: NULLIF() returns NULL when NULL is the first argument
+--source ../include/have_columnstore.inc
+
+--disable_warnings
+DROP DATABASE IF EXISTS mcol5992;
+--enable_warnings
+
+CREATE DATABASE mcol5992;
+USE mcol5992;
+
+CREATE TABLE `CS_LEG` (`ID` bigint(20) DEFAULT NULL, `sess_count` smallint(6) DEFAULT NULL) ENGINE=Columnstore;
+CREATE TABLE `INNO_LEG` (`ID` bigint(20) DEFAULT NULL, `sess_count` smallint(6) DEFAULT NULL) ENGINE=InnoDB;
```

Comment:
This table is not used in the test, maybe remove it in order to reduce the CO₂ footprint? 8))